tests/integration/fresh/fresh_test.go

```diff
@@ -21,7 +21,7 @@ func TestFreshDeployment(t *testing.T) {
 	noCleanup := os.Getenv("NOCLEANUP") == "true"
 
 	// Validate on various kubernetes versions
-	for _, k8sVersion := range []string{"1.19", "1.20", "1.21"} {
+	for _, k8sVersion := range []string{"1.19"} {
```

Comment:
Temp change for testing?

Reply:
Yeah, just to cut down on the churn whilst testing.